### ISSUE TITLE
[Feature suggestion] Icon Widget(s): add text-shadow control

### ISSUE THREAD

**Issue by VerdiH** [elementor/elementor#5579]
Nothing important, just nice to have.

### What problem is your feature request going to solve? Please describe.

- Same shadow settings for icons as for the buttons and headings that are next to them.
- Consistency.

### Describe the solution you'd like

There are several widgets where you can use icons. In none of them are you able to add a shadow using the UI.

### Describe alternatives you've considered

Custom CSS

**Comment by reelism** [comment]
+1

Seems an oversight. 

Any plans?

**Comment by Ver3** [comment]
Altered the title and text a bit to clarify. Just ran into the 'problem' again today. Went here to post it, then found I already did :-)

**Comment by nbi1** [comment]
Yes, this is extremely annoying. This issue also surfaces for the icon list widget. The problem is even more serious if the list inherits the shadow setting from a parent - in that scenario you can't get rid of the shadow which renders the widget unusable. As a work around I now need to find custom css to turn off the shadow or create an icon list from scratch. A serious design gaffe on part of Elementor.

**Comment by VerdiH** [comment]
> Yes, this is extremely annoying. This issue also surfaces for the icon list widget. The problem is even more serious if the list inherits the shadow setting from a parent - in that scenario you can't get rid of the shadow which renders the widget unusable. As a work around I now need to find custom css to turn off the shadow or create an icon list from scratch. A serious design gaffe on part of Elementor.

Hi nbi1, personally I do not think it's a serious matter all. Not even some thing for high up a wish list (I already have centralized design settings up there ;-) 

This minor issue is easily fixed with a little CSS. It would just save a bit of time and be slightly more consistent if taken care of. I;m sure they will at some point.

**Comment by nbi1** [comment]
I couldn't disagree more. First of all I've only been using Elementor 
for a short time and I've already discovered a pile of problems - how 
long will that list be once I've used it for more than a few days?? 
Secondly Elementor Pro is not free - I expect a commercial package not 
to be riddled with such obvious oversights. If I need to use css then 
why am I using a widget/control based designer, why not just use 
HTML/javascript and not have to be concerned about what goes on under 
the covers?
